res/values/strings.xml

```diff
@@ -169,7 +169,7 @@
     <string name="menu_zoom_out">Zoom out</string>
     <string name="menu_save_log">Save log</string>
     <string name="title_share_location">Share location with all group members</string>
-    <string name="device_talk">Device Messages</string>
+    <string name="device_talk">Device messages</string>
```

Comment:
i think names can be left in uppercase completely, so "Device Messages" should be fine here.

Reply:
All strings lowercase at the moment. Regularity and consistency are more important.

Reply:
fair enough, i do not have a strong opinion in that.